core/web/common.go

```diff
@@ -9,9 +9,9 @@ import (
 )
 
 var (
-	ErrMissingChainID = errors.New("evmChainID does not match any local chains")
-	ErrInvalidChainID = errors.New("invalid evmChainID")
-	ErrMultipleChains = errors.New("more than one chain available, you must specify evmChainID parameter")
+	ErrMissingChainID = errors.New("chain --id does not match any local chains")
+	ErrInvalidChainID = errors.New("invalid chain --id")
+	ErrMultipleChains = errors.New("more than one chain available, you must specify (evm chain) --id parameter")
```

Comment:
These errors are not exclusive to the CLI, since they are part of the web service. We should use language here that is neutral and works for anything that interfaces with this service. Maybe just `id` or `chain id`, without assuming anything about it being a flag or http/json param.

Reply:
Thanks, @jmank88. I have moved from `--id` flag format to more general `id` that could fit different contexts.